.ruby-gemset

```diff
@@ -0,0 +1 @@
+consul-cookbook
```

Comment:
Certanly, sorry! Would it be fine if I add them to .gitignore instead, so that I can use them locally?